guides/source/security.md

```diff
@@ -666,7 +666,7 @@ Project.where("name = '#{params[:name]}'")
 
 And now let's inject another query using the UNION statement:
 
-```
+```sql
```

Comment:
This technically isn't valid SQL by itself, so the highlighting doesn't really work here.